src/plugins/shared_dictionaries_plugin/shared_dictionaries_column_processor.hpp

```diff
@@ -0,0 +1,86 @@
+#pragma once
+
+#include "hyrise.hpp"
+#include "shared_dictionaries_plugin.hpp"
+#include "storage/table.hpp"
+#include "utils/size_estimation_utils.hpp"
+
+namespace opossum {
+
+/**
```

Comment:
I could imagine some higher level documentation of the process to be helpful here.

Reply:
Fixed in https://github.com/hyrise/hyrise/pull/2394/commits/7f88317d0867f9bfd92688f5061daea73a39be35